src/Sylius/Bundle/AdminBundle/Resources/config/routing/ajax/product_variant.yml

```diff
@@ -33,10 +33,12 @@ sylius_admin_ajax_all_product_variants_by_phrase:
             serialization_groups: [Autocomplete]
             permission: true
             repository:
-                method: findByPhrase
+                method: findByPhrasePartial
                 arguments:
                     phrase: $phrase
                     locale: expr:service('sylius.context.locale').getLocaleCode()
+                    page: 1
```

Comment:
Can we simply add optional argument `limit`? and make it in a similar way like in `Sylius\Bundle\ProductBundle\Doctrine\ORM\ProductRepository::findByNamePart`

you are adding pagination, but it isn't mentioned in task at all.

Reply:
Agreed 👍  we can just make a simple limit without pagination and it would already be an improvement 🚀

Reply:
Is anyone going beyond the first page of Google Search? 🤷‍♂️ 
If it's on on the first one, then your query is wrong and you change it. 😆